legal-api/src/legal_api/reports/report.py

```diff
@@ -899,6 +905,49 @@ def _format_change_of_registration_data(self, filing, filing_type):  # noqa: E50
             parties_deleted = [p for p in existing_party_json if p['officer']['id'] not in parties_to_edit]
             filing['ceasedParties'] = parties_deleted
 
+    def _format_continuation_in_data(self, filing):
+        self._format_address(filing['continuationIn']['offices']['registeredOffice']['deliveryAddress'])
+        self._format_address(filing['continuationIn']['offices']['registeredOffice']['mailingAddress'])
+        if 'recordsOffice' in filing['continuationIn']['offices']:
+            self._format_address(filing['continuationIn']['offices']['recordsOffice']['deliveryAddress'])
+            self._format_address(filing['continuationIn']['offices']['recordsOffice']['mailingAddress'])
+        self._format_directors(filing['continuationIn']['parties'])
+        # create helper lists
+        filing['nameRequest'] = filing['continuationIn'].get('nameRequest')
+        filing['listOfTranslations'] = filing['continuationIn'].get('nameTranslations', [])
+        filing['offices'] = filing['continuationIn']['offices']
+        filing['parties'] = filing['continuationIn']['parties']
+        if filing['continuationIn'].get('shareClasses', None):
+            filing['shareClasses'] = filing['continuationIn']['shareClasses']
+        elif 'shareStructure' in filing['continuationIn']:
+            filing['shareClasses'] = filing['continuationIn']['shareStructure']['shareClasses']
+
+        # set expro business
+        filing['mode'] = filing['continuationIn'].get('mode')
```

Comment:
cannot dependent on mode in filing json as its not an expected value in schema. use filing['exproBusiness'] to identify EXPRO

Reply:
got it!